packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -282,7 +282,7 @@ impl Experiment {
             .wrap_err("Could not send `Init` message")?;
         debug!("Sent init message to \"{experiment_name}\"");
 
-        let mut graceful_finish = true;
+        let mut graceful_finish = EngineExitStatus::Success;
```

Comment:
Can we rename the variable to sth. like `engine_exit_status`?

Reply:
Renamed https://github.com/hashintel/hash/pull/477/commits/400218c8a1647e9b95c1e8549c139ea05b2b73fd